### PR TITLE
enhance!: modify CI environment value to true

### PR DESCRIPTION
**Pull request by dtanner**
https://github.com/go-vela/community/issues/554

### CODE REVIEW

File: compiler/native/environment_test.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ package native
 
 import (
 	"flag"
+	"github.com/urfave/cli/v2"
```
Comment (cognifloyd) [inline compiler/native/environment_test.go:9]: nit: revert import order change.

**Review by cognifloyd — APPROVED** [review]
I have a minor nit, but otherwise LGTM.

File: compiler/native/environment_test.go (hunk 1 above)
Comment (wass3r) [reply compiler/native/environment_test.go:9]: yea, we should configure one of the linters to catch this for us. i think we have discrepancies on import order in a few places.

**Review by wass3r — APPROVED** [review]

**Review by ecrupper — APPROVED** [review]
LGTM

**Review by wass3r — APPROVED** [review]

**Pull request merged**